.test-env

```diff
@@ -1,6 +1,6 @@
 # Configs for testing repo download:
 SDK_TESTING_URL="https://github.com/algorand/algorand-sdk-testing"
-SDK_TESTING_BRANCH="master"
+SDK_TESTING_BRANCH="disassemble"
```

Comment:
Leaving myself a visual reminder to reset `SDK_TESTING_BRANCH` prior to merge.

```suggestion
SDK_TESTING_BRANCH="master"
```

Reply:
Since we [merged and deleted this branch in the SDK repo](https://github.com/algorand/algorand-sdk-testing/pull/251), I think it makes sense to commit this.